bundles/org.openhab.binding.wizlighting/src/main/java/org/openhab/binding/wizlighting/internal/enums/WizLightingModuleType.java

```diff
@@ -0,0 +1,62 @@
+/**
+ * Copyright (c) 2010-2020 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.wizlighting.internal.enums;
+
+import static org.openhab.binding.wizlighting.internal.WizLightingBindingConstants.*;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import org.eclipse.smarthome.core.thing.ThingTypeUID;
+
+/**
+ * This enum represents the possible scene modes.
+ *
+ * @author Sara Geleskie Damiano - Initial contribution
+ *
+ */
+public enum WizLightingModuleType {
+    fullColorWifi("ESP01_SHRGB1C_31", THING_TYPE_WIZ_COLOR_BULB),
```

Comment:
The standard enum naming convention is that enum values should start with a capital letter.

Reply:
Fixed